src/plugins/event_annotation/public/components/group_editor_controls/annotation_list.tsx

```diff
@@ -110,7 +108,7 @@ export const AnnotationList = ({
           getAdditionalClassesOnDroppable={
             DropTargetSwapDuplicateCombine.getAdditionalClassesOnDroppable
           }
-          dropTypes={dragging ? ['field_add'] : []}
```

Comment:
Previously it was necessary for reorder styles to be applied only after drag started. Was this behaviour changed?

Reply:
Yes, it's from the PR before this one - the styles are applied inside the DragDrop component where we check for `dragging` so no need to do the check here as we only have one type of dropTypes for annotations.

Reply:
I like it.